card/mockCard.go

```diff
@@ -758,6 +837,78 @@ func (c *MockCard) SendPhonons(keyIndices []uint16, extendedRequest bool) (trans
 // 	return nil
 // }
 
+func (c *MockCard) PostPhonons(recipientsPublicKey []byte, nonce uint64, keyIndices []uint16) (transferPhononPackets []byte, err error) {
+	log.Debug("sending mock POST_PHONONS command")
+	var outgoingPhonons []byte
+
+	for _, k := range keyIndices {
+		if int(k) >= len(c.Phonons) {
+			return nil, errors.New("keyIndex exceeds length of phonon list")
+		}
+		if c.Phonons[k].deleted {
+			return nil, errors.New("cannot access deleted phonon")
+		}
+
+		phonon := c.Phonons[k]
+
+		iv := util.RandomKey(16)
```

Comment:
This is not really an iv (init vector), since that is a value used to establish a starting value during a rotating cipher like AES, and this value is used to provide a single randomizing factor during the encryption of a single value. I would call this a salt instead to avoid confusion.